docs/common/craft-parts/reference/plugins/ruby_plugin.rst

```diff
@@ -0,0 +1,125 @@
+.. _craft_parts_ruby_plugin:
+
+Ruby plugin
+===========
+
+The Ruby plugin can be used in projects that need to bundle a specific
+version and variant of the Ruby interpreter within the final artifact. The
+plugin can further be used to install individual gems or trigger a build tool
+such as rake or bundler.
+
+.. _craft_parts_ruby_plugin-keywords:
+
+Keys
+----
+
+This plugin provides the following unique keys.
+
+
+ruby-gems
+~~~~~~~~~
+
+**Type:** list of strings
+
+List of gems to be installed.
+
+
+ruby-use-bundler
+~~~~~~~~~~~~~~~~
+
+**Type:** boolean
+
+**Default:** False
+
+When set to ``true``, the plugin runs the ``bundle`` command as part of the
+``build`` step.
+
+
+ruby-flavor
+~~~~~~~~~~~
+
+**Type:** string
+
+**Default:** ruby
+
+Specifies which implementation of the Ruby interpreter the plugin
+should build. Supported values are ``ruby``, ``jruby``, ``rbx``,
+``truffleruby``, and ``mruby``.
```

Comment:
```suggestion
The Ruby interpreter to compile and include. The supported interpreters are:

- ``ruby``
- ``jruby``
- ``rbx``
- ``truffleruby``
- ``mruby``
```

I don't believe we need to keep the word "implementation", but I'm not very familiar.